### DISCUSSION TITLE
Ich bekomme keine Verbindung zum VLC

### DISCUSSION THREAD

**Discussion by Stefanh74** [GerryDazoo/Slinger#279] [Announcements]
Version : 3.08d Running on Windows-10-10.0.19045-SP0 pid= 10024
Using config file config.ini
Connection Manager Running on port 8080 with 1 max streams using URL Gerry.
BOXES [('sb1', 'M1')]
Streamer Running:  1 config.ini M1 M1 8080 8388608
starting up on  port 8080 8388608
Checking for slingbox at ('192.168.0.193', 5304)
('192.168.0.193', 5304) OK
Slingbox M1 Using slingbox at  ('192.168.0.193', 5304)
Streamer:  Slingbox M1 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
Building page for M1
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
Hacker Alert. Invalid Request from  ('127.0.0.1', 55761)
GET /slingbox HTTP/1.1
Host: 127.0.0.1:8080
Accept: */*
Accept-Language: de
User-Agent: VLC/3.0.12 LibVLC/3.0.12
Range: bytes=0-

Hacker Alert. Invalid Request from  ('127.0.0.1', 55762)
GET /slingbox HTTP/1.0
Host: 127.0.0.1:8080
User-Agent: VLC/3.0.12 LibVLC/3.0.12
Icy-MetaData: 1

Was mache ich falsch?

**Comment by jkaponis** [comment]
try vlc http://127.0.0.1:8080/Gerry/M1

**Comment by GerryDazoo** [comment]
Thanks jkaponis, That is the correct answer based on what we see in the posted logs. That unified_config.ini file was only supposed to be a sample. Who knew that people would copy it verbatim. I guess I've famous now with everybody using my name in their URLs.  
Do you need a job answering this question for all the other people that can't figure it out?

**Comment by Stefanh74** [comment]
Jetzt bekomme ich diese Meldung  

Slinginfo  "350/500/M1" 12 30 192.168.0.193 5304 3072 10
Slingbox M1 Starting Stream for  127.0.0.1:51529
Connecting... ('192.168.0.193', 5304) Control
Slingbox M1 Selecting Video Source 0
Keycodes= [1, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 21, 24, 25, 26, 27, 28, 29, 33, 35, 36, 38, 39, 40, 41, 42, 46, 62, 63, 64, 65, 66, 67, 68, 69, 70, 74, 75, 76, 77, 78]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.0.193', 5304) Stream
Slingbox M1 Stream started at 11/13/2022, 09:59:01.921  744 0
11/13/2022, 09:59:05.166  Slingbox M1 Stream Terminated for  127.0.0.1:51529 [WinError 10053] Eine bestehende Verbindung wurde softwaregesteuert
durch den Hostcomputer abgebrochen
Slingbox M1 Shutting down connections
Slingbox M1 Logging Out
Streamer:  Slingbox M1 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

**Comment by waldorf-statler** [comment]
@Stefanh74 this looks very similar to a problem I had earlier on with Windows VLC. The solution was to uninstall VLC and install the latest version. There is some known caching problem with some earlier versions. However of course it could be something else entirely.

**Comment by Stefanh74** [comment]
Ja das war das Problem den neuen VLC installiert und jetzt Läuft im lokalen Netzwerk  
Jetzt versuche ich mich nach an eine Externe Verbindung

**Comment by Stefanh74** [comment]
Ich habe noch zwei weitere Boxen eine 350 und M1 
Ist es noch möglich das Passwort der Boxen auszulesen?
Oder kann ich diese entsorgen

**Comment by waldorf-statler** [comment]
There is a lot of information online - maybe too much - but it seems some people managed to get the machines going with a hard reset and password "admin"; but this seems risky, and not sure it is working after Nov 9. 

However if you have slingbox files in your pc (e.g. AppData/Roaming/Sling Media) you might be in luck. For example, I can still find mine in a file "console.log". Basically check any files and if anything has text, search for "password" and you might find gold...

**Comment by Stefanh74** [comment]
11/14/2022, 21:58:51.613  Registering Slingboxes {'9CB2E309ECE8CECB4510D2623EEE52EF': 8080}
Getting external IP address
Redirector, my external IP address 46.223.10.139
11/14/2022, 21:58:53.981  PING failure ('46.223.10.139', 8080) [WinError 10061] Es konnte keine Verbindung hergestellt werden, da der Zielcomputer die Verbindung verweigerte
No working external portmaps. Not registering slingboxes
Retry 1 in 10 minutes

Was muss ich noch für den Zugriff von außen einstellen ?

**Reply by GerryDazoo** [reply]
Don't worry about that for now. Comment out your FinderID in the config.ini. The error you're seeing is not service affecting. You should be able to stream successfully. The latest version of the software has changed these messages to be more informative after many people have mistoke them for a real problem.
